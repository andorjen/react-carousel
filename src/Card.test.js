import React from "react";
import {render} from "@testing-library/react";
import Card from "./Card";

it('renders card without crashing', function(){
    render( <Card caption="test" src="http://localhost:3000/static/media/image1.08e1519e.jpg" currNum={1} totalNum={3} />)
})

it("matches snapshot", function(){
    const {container} = render( <Card caption="test" src="http://localhost:3000/static/media/image1.08e1519e.jpg" currNum={1} totalNum={3} />);
    expect(container).toMatchSnapshot();
})