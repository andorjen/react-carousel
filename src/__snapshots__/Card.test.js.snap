// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="http://localhost:3000/static/media/image1.08e1519e.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
